lexicon/conversion.py

```diff
@@ -280,3 +279,109 @@ def run(self):
         with uopen(self.out_bliss_lexicon.get_path(), "w") as lexicon_file:
             lexicon_file.write('<?xml version="1.0" encoding="utf-8"?>\n')
             lexicon_file.write(ET.tostring(lex.to_xml(), "unicode"))
+
+
+class SpellingConversionJob(Job):
+    """Convert lexicon to a new one with other regional spellings
+    e.g. US -> GB spelling
+    """
+
+    def __init__(
+        self,
+        bliss_lexicon,
+        orth_mapping_file,
+        reverse_mapping=False,
+        mapping_delimiter=" ",
```

Comment:
maybe better default would be "any whitespace character" to also incorporate tabs and the like. you may decide.